docs/quickstart.rst

```diff
@@ -42,42 +42,87 @@ following command.
 
 .. code-block:: shell
 
-   $ python setup.py install
+   $ pip install -e .
```

Comment:
If this is for installing, the `-e` should be removed.  `-e` is for development.